libdevcore/Common.cpp

```diff
@@ -113,10 +113,8 @@ that users do not need to install language packs for their OS.
 void setDefaultOrCLocale()
 {
 #if __unix__
-    if (!std::setlocale(LC_ALL, ""))
```

Comment:
Why is this change needed?

Reply:
In libc++ the `setlocale` is not in the `std` namespace.